src/components/CreateTweet.jsx

```diff
@@ -1,23 +1,49 @@
-import { useState } from 'react'
+import { useContext, useState } from 'react'
 import imgDoge from '../assets/images/doge.jpg'
+import { AppContext, ThemeContext } from '../App'
 
-export default function CreateTweet({ tweets, setTweets, user, theme }) {
-    const [content, setContent] = useState('')
+const Initial_Post = () => {
+    const content = localStorage.getItem('content')
+
+    return {
+        content: content || ''
```

Comment:
This is a nitpick, but it would be better if you did the `|| ''` when you get the value from localstorage. Like so
```js
const content = localStorage.getItem('content') || ''
```

That way, you know `content` always has a correct value, and you don't have to do the `|| ''` whenever you want to use the variable.